<launch>
    <node name="background_averager" pkg="background_filters" type="background_sampler" output="screen" >
        <remap from="image" to="camera/image_color" />
        <param name="number_of_samples" value="100" />
        <param name="sampling_delay" value="0.0" />
    </node>
</launch>
